docs/python/debugging.md

```diff
@@ -125,25 +328,13 @@ You can also rely on a relative path from the workspace root. For example, if th
 "program": "${workspaceFolder}/pokemongo_bot/event_handlers/__init__.py",
 ```
 
-### `pythonPath`
+### `python`
 
 Points to the Python interpreter to be used for debugging, which can be a folder containing a Python interpreter. The value can use variables like `${workspaceFolder}` and `${workspaceFolder}/.venv`.
```

Comment:
That bit about "a folder containing a Python interpreter" is not entirely clear to me. The debugger itself definitely requires this property to be a full path to the interpreter binary itself - i.e. "C:\Python\python.exe", not just "C:\Python". But maybe VSCode is doing something else here... @karthiknadig, do you recall if there are any provisions in debug adapter factory etc for pointing at folders like that?

Reply:
No. It should be full path to the interpreter binary.

Reply:
Ok, corrected.